bagua/torch_api/algorithms/async_model_average.py

```diff
@@ -229,35 +229,56 @@ def _run_async_loop(self, bagua_module: BaguaModule):
             comm_step += 1
             time.sleep(self.sync_interval_ms / 1000)
 
-    def abort(self, bagua_module: BaguaModule):
+    def abort(self, bagua_ddp: Union[BaguaDistributedDataParallel]):
```

Comment:
Union seems redundant here.

Reply:
It is union, but some others union can't write here. But because of the circular dependency, some other things can’t be written here